### PR TITLE
Add infos on manage your locales.md

### PR DESCRIPTION
**Pull request by LaureBro**
added sub cat: ## Format of the new locale to import with the format requested and the limitation of 20 char.
Added ## Steps to add a new locale
remove the 1st warning block and added it at the end of the steps. t the warning, since it's possible to now add them in the channel field.
add a ee block on locales permissions
add a warning regarding locale deletion (they can't be removed from UI)

### CODE REVIEW

File: content/md/first-steps/manage-locales.md [hunk 1]
```diff
@@ -30,15 +30,28 @@ Locales which are not used in at least one channel are shown as inactive.
 # How to add a new locale?
 
 You are missing a locale? It is very simple to add a new locale using imports.  
+
+## Format of the new locale to import
+
+Your new locales have to respect one of the following formats: xx_YY, xxx_YY, xx_YY_ZZ... never upload a locale with empty spaces in it. If you upload a `es_DE` locale, it will appear as `Spanish (Germany)`, if you upload a locale called `English_test_web` it will appear as `English (test, Web)`. 
+
+The PIM limits to 20 the number of characters that can be used for a locale.
```
Comment (NolwennP) [inline content/md/first-steps/manage-locales.md]: for a locale name

File: content/md/first-steps/manage-locales.md [hunk 2]
```diff
@@ -30,15 +30,28 @@ Locales which are not used in at least one channel are shown as inactive.
...
+
+## Steps to add a new locale
+
 For instance, to add a `en_HK` locale, follow these steps:
 1.  Create an import XLSX or CSV file containing two values: a locale `code` column like below:  
   `code`  
   `en_HK`  
 1.  Import this file using an import job (`csv_locale_import` or `xlsx_locale_import`). Once the file is imported, the new locale will appear under `Settings/Locales`.
-1.  To enable it and add it to a channel, use a channel import job (again csv or xlsx) and add the `en_HK` code in the `locales` column.
+1.  To enable it and add it to a channel, go to `Settings/Channels` and add it in the dedicated field. If you cannot find it use a channel import job (csv or xlsx) and add the `en_HK` code in the `locales` column.
```
Comment (NolwennP) [inline content/md/first-steps/manage-locales.md]: If you cannot find it, use

File: content/md/first-steps/manage-locales.md [hunk 3]
```diff
@@ -30,15 +30,28 @@ Locales which are not used in at least one channel are shown as inactive.
...
 
 And that's it! Your `en_HK` locale is activated for your channel!
 
+:::ee
+If you do not see your new locales in the User Interface, it's maybe because you forgot to set permissions on it in `Settings/Locales`. Select your locale and go under the `Permissions` tab. To know how to set up permissions on locales, check the [Set Rights depending on the locale](/articles/access-rights-on-products.html#rights-depending-on-the-locale) article.
+:::
+
 :::warning
-You can only enable your new locale with a channel import. It will not work using the user interface.
+You can add locales easily, but you cannot remove them afterwards from the user interface. Please make sure you really need it before uploading it!
```
Comment (NolwennP) [inline content/md/first-steps/manage-locales.md]: `easily add locales`?

**Comment by LaureBro** [comment]
I have made some changes on the article, I have renamed it as Manage your locales as you suggested, it's better. 
Also, I have removed the 1st part, the overview part, imho it was only messing up with our brains ("The locales are used to set values for attributes which may differ according to the relevant language/locale (for examples English UK (en_GB), English US (en_US), English AU (en_AU))."), and it was saying the same things as /articles/what-is-a-locale.html article. 
Maybe we could add at the top "If you are not familiar with the concept of locale, please read the "what-is-a-locale" article before.

File: content/md/first-steps/manage-locales.md [hunk 4]
```diff
@@ -1,44 +1,58 @@
...
+:::
+
 :::warning
-You can only enable your new locale with a channel import. It will not work using the user interface.
+You can easily add locales, but you cannot remove them afterwards from the user interface. Please make sure you really need additional locales before uploading them!
 :::
+
+# Set permissions on locales
+
+Like on Attribute groups and on Categories, you can define specific user permissions on locales. Check the [Set Rights depending on the locale](/articles/access-rights-on-products.html#rights-depending-on-the-locale) article to know how to add/remove permissions on locales.
```
Comment (NolwennP) [inline content/md/first-steps/manage-locales.md]: Why did you use uppercase on `Attribute groups` and `Categories`?

**Pull request merged**